app/views/groups/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<section class='transactions'>
+  <nav>
+    <ul>
+    <li>
+    <%= link_to user_groups_path(@user) do %>
+      <svg class="navBtn" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-menu"><line x1="4" x2="20" y1="12" y2="12"/><line x1="4" x2="20" y1="6" y2="6"/><line x1="4" x2="20" y1="18" y2="18"/></svg>
+    <% end %>
```

Comment:
- [ ] The back button should be "<" according to the design and requirement, not the hamburger menu display you have now.

  ![image](https://github.com/78fatemanazari/ezPay-rails-app/assets/54848363/1e3d4764-2f7c-4db2-b7e3-97e213a09064)

Reply:
Ok, I will make it, Thanks